_data/cryptocurrencies.yml

```
@@ -335,6 +335,7 @@ websites:
       - sms
       - proprietary
       - phone
+      - hardware
```

```suggestion
      - hardware
      - u2f
```
FIDO U2F is supported.